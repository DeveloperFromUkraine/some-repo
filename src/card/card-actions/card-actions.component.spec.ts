import { ComponentFixture, TestBed } from "@angular/core/testing";
import { CardActionsComponent } from "./index";
import { ComponentTest } from '../../../test/test-bed/component';

describe('CardActionsComponent', () => {
    let fixture: ComponentFixture<CardActionsComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [CardActionsComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(CardActionsComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
}); 