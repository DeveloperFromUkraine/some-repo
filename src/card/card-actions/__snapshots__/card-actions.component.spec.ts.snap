// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardActionsComponent should match snapshot 1`] = `
<ign-card-actions>
  
</ign-card-actions>
`;
